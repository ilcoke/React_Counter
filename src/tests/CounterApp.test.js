import React from 'react';
import "@testing-library/jest-dom"
import {shallow} from 'enzyme'
import CounterApp from '../CounterApp ';


describe('Revisa componente primer app', () => {
    test('should show <primerApp /> whith 0 defoult', () => {
        const wrapper = shallow(<CounterApp />);
        expect( wrapper ).toMatchSnapshot()
    })
    test('should show <PrimerApp /> with 100 ', () => {
        const wrapper = shallow(<CounterApp value = {100}/>);
        const findValue = wrapper.find('h2').text().trim();
        expect(findValue).toBe('100')
    })
    
})
