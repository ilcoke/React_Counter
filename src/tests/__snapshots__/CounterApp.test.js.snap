// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Revisa componente primer app should show <primerApp /> 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
    10
  </h2>
  <button
    onClick={[Function]}
  >
     +1 
  </button>
  <button
    onClick={[Function]}
  >
     reset
  </button>
  <button
    onClick={[Function]}
  >
     -1 
  </button>
</Fragment>
`;

exports[`Revisa componente primer app should show <primerApp /> whith 0 defoult 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
    10
  </h2>
  <button
    onClick={[Function]}
  >
     +1 
  </button>
  <button
    onClick={[Function]}
  >
     reset
  </button>
  <button
    onClick={[Function]}
  >
     -1 
  </button>
</Fragment>
`;
